dotnet/src/Planners/Planners.Handlebars/Handlebars/HandlebarsTemplateEngineExtensions.cs

```diff
@@ -27,16 +27,14 @@ internal sealed class HandlebarsTemplateEngineExtensions
     /// Renders a Handlebars template in the context of a Semantic Kernel.
     /// </summary>
     /// <param name="kernel">The Semantic Kernel.</param>
-    /// <param name="contextVariables">The execution context variables.</param>
     /// <param name="template">The Handlebars template to render.</param>
-    /// <param name="variables">The dictionary of variables to pass to the Handlebars template engine.</param>
+    /// <param name="arguments">The dictionary of arguments to pass to the Handlebars template engine.</param>
     /// <param name="cancellationToken">The cancellation token.</param>
     /// <returns>The rendered Handlebars template.</returns>
     public static string Render(
         Kernel kernel,
-        ContextVariables contextVariables,
         string template,
-        Dictionary<string, object?> variables,
+        Dictionary<string, object?> arguments,
```

Comment:
Should some arguments that are currently dictionaries becomes KernelFunctionArguments?

Reply:
I believe so, as mentioned in several other comments, except for this specific case, which will be refactored later when we have support for complex types.